src/main/java/com/example/music/backend/verification/service/VerificationTokenService.java

```diff
@@ -0,0 +1,12 @@
+package com.example.music.backend.verification.service;
+
+import com.example.music.backend.user.domain.User;
+import com.example.music.backend.verification.domain.VerificationToken;
+
+public interface VerificationTokenService {
+
+    void createVerificationToken(User user, String token);
```

Comment:
Question from previous PR.
"I expect that method with "Create" return something, but actually it's "void""

Reply:
fixed